docs/topics/wasm/wasm-troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+[//]: # (title: Troubleshooting)
+
+Most of the browsers support WebAssembly. For recent browser versions, WebAssembly works by default. However,
+for older browser versions, you still need to update the browser settings.
+
+To run a Kotlin/Wasm project, check if you need to update the settings of the target environment:
+
+<tabs>
+<tab title="Chrome">
+
+* For version 119 or later:
+  
+  It works by default.
+
+* For older versions:
```

Comment:
For older browsers, it also will require using the older Kotlin toolchain (<=1.9.20-Beta), since Kotlin 1.9.20-Beta2 we generates final opcodes.

Reply:
Maybe for older browsers, it would be enough to have a generic comment like:
"For older browsers, make sure that Wasm GC is supported and turned on and you use Kotlin toolchain before 1.9.20"

Reply:
-Kotlin 1.9.20: Added. 
-Regarding the generic comment, we wouldn't like to leave users with open questions. In this sense, if we tell them to make sure Wasm is turned on, it would be helpful to also tell them how to turn it on. So, we could leave the 3 steps; they're concise and don't add noise to the information flow. What do you think?

Reply:
ok, thanks